File: src/sdk-reference/cpp/1/collection/validate-specifications/snippets/validate-specifications.cpp
try {
  std::string specifications = R"({
    "strict": false,
    "fields": {
      "license": {
        "mandatory": true,
        "type": "string"
      }
    }
  })";

  kuzzleio::validation_response *validation_response =
    kuzzle->collection->validateSpecifications(
      "nyc-open-data",
      "yellow-taxi",
      specifications);

  if (validation_response->valid) {
    std::cout << "Specifications are valid" << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/collection/validate-specifications/snippets/validate-specifications.cs
try {
  string specifications = @"{
    ""strict"": false,
    ""fields"": {
      ""license"": {
        ""mandatory"": true,
        ""type"": ""string""
      }
    }
  }";

  validation_response validation_response =
    kuzzle.collection.validateSpecifications(
      "nyc-open-data",
      "yellow-taxi",
      specifications);

  if (validation_response.valid) {
    Console.WriteLine("Specifications are valid");
  }
} catch  {
  Console.Error.WriteLine(e.what());
}
